StreamNet/Extensions/StringExtensions.cs

```diff
@@ -3,5 +3,6 @@ namespace StreamNet.Extensions
     public static class StringExtensions
     {
         public static bool IsNullOrEmpty(this string str) => string.IsNullOrEmpty(str);
+        public static bool IsNullOrWhitespace(this string str) => string.IsNullOrWhiteSpace(str);
```

Comment:
The idea of this suggestion is copy the [string.isBlank() from Java 11](https://www.baeldung.com/java-blank-empty-strings#string-trim).
```suggestion
        public static bool IsBlank(this string str) => string.IsNullOrEmpty(str.Trim());
```